src/buffer/out/textBuffer.cpp

```diff
@@ -952,7 +952,7 @@ Microsoft::Console::Render::IRenderTarget& TextBuffer::GetRenderTarget() noexcep
 // - includeCharacterRun - include the character run located at the beginning of the word
 // Return Value:
 // - The COORD for the first character on the "word"  (inclusive)
-const COORD TextBuffer::GetWordStart(const COORD target, const std::wstring& wordDelimiters, bool includeCharacterRun) const
+const COORD TextBuffer::GetWordStart(const COORD target, const std::wstring_view wordDelimiters, bool includeCharacterRun) const

```

Comment:
Oh. Well. I made a comment about this a thousand times in the other PR. Sorry, I should have looked here first.